### DISCUSSION TITLE
About missing features

### DISCUSSION THREAD

**Discussion by 2bbb** [2bbb/ofxPubSubOsc#46] [Ideas]
Feel free to discussion!

**Comment by moebiussurfing** [comment]
what about support for ofParameter < void > ?
we can use a bool as a trigger settling it as true and then to false; but this has some "tricky behaviors":

1. when adding a publisher for a bool, to be used as a bang trigger, it trigs true but the (undesired) false too.
Maybe can be solved using the conditionals like ofxPublishOscIf (?)
2. if we set that bool to false on the same frame, is nothing is sent (true neither false), bc the publisher sending happens after update()
so to avoid that we need to introduce a flag/state/timer.

**Reply by moebiussurfing** [reply]
Trying to send only when a bool param changed to true,
I tried to do it like that:

```.cpp
ofParameter<bool> p { "bang", false };
// set to true then to false after some time.

// set publisher to send the bang
auto condition = [&]() {
  return p.get() == true;
};
ofxPublishOscIf(condition, OutputIp, OutputPort, address, p);
```

But what I get is that if the bool param is true, it sends a message every frame while it being true,
and stop sending when goes to false.

There's some similar way to send only when changed to true?

**Comment by moebiussurfing** [comment]
Hello, 
I am looking for a way to disable a receiver/subscriber, but per address.

I would like to avoid to pass again the ofParameter/var associated, 
just like we can do for the sender/publisher:

It's possible like we can do for publishers but for the subscribers instead?

```.cpp
ofxGetOscPublisher("localhost", 22222)->stopPublishTemporary("/hoge");
ofxGetOscPublisher("localhost", 22222)->resumePublish("/hoge");
```
